src/h5web/vis-packs/core/complex/config.tsx

```diff
@@ -1,7 +1,7 @@
 import create from 'zustand';
 import { persist } from 'zustand/middleware';
 import type { ConfigProviderProps } from '../../models';
-import createContext from 'zustand/context';
+import createContext from 'zustand/context.js';
```

Comment:
In adding support for ES modules in Node, zustand broke our imports... Until CRA switches to `webpack@5`, we have to load the CommonJS module `context.js` explicitly (instead of the ES module `context.mjs`, which is the default when no extension is specified).

Note that this is not a problem with `zustand/middleware`, as this module doesn't `import` any other module.

Reply:
They rolled back some of the changes in 3.5.10: https://github.com/pmndrs/zustand/releases/tag/v3.5.10.

You might want to see if this is still needed if we update to 3.5.10 instead of 3.5.9

Reply:
Ah yes, saw this. Thanks for the reminder.